dev/com.ibm.ws.jaxrs.2.0_fat/publish/servers/com.ibm.ws.jaxrs.fat.restmetrics/server.xml

```diff
@@ -0,0 +1,15 @@
+<server>
+    <featureManager>
+        <feature>jaxrs-2.0</feature>
+        <feature>mpMetrics-2.0</feature>
+        <feature>monitor-1.0</feature>
+        <feature>jaxrsMonitor-1.0</feature>
+    </featureManager>
+
+    <mpMetrics authentication = "false"/>
+
+
+  	<include location="../fatTestPorts.xml"/>
+  	
+  	<javaPermission className="javax.management.MBeanServerPermission" name="createMBeanServer"/>
```

Comment:
Why do you need this? Liberty shouldn't need permission to create the MBeanServer, and I don't see that the test application is creating one.  (If Liberty _does_ need permission to create the MBeanServer, then that is a bug that needs to be fixed).

Reply:
Discussed this with Andy and decided to instead add the following comment:
<!-- Test application requires this permission in order to acquire access to Liberty's MBean server -->